docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+
+services:
+  web:
+    build: ../archivist-web/
```

Comment:
Can you update this to just `.` since it is only referring to the current directory really?  

Alternatively, we can ditch the docker-compose file here and just build it normally using the `docker` command -- I copied the docker-compose over for the api because we need to spin up mongo as well.